### ISSUE TITLE
handling notification sounds

### ISSUE THREAD

**Issue by arnaudjuracek** [kyleneideck/BackgroundMusic#11]
*First of all : you rocks, this utility is gonna be awesome.*

Now my suggestion : currently the utility auto-pauses the player when any sound is played elsewhere. The problem being that when receiving a notification on —saying— Facebook on Chrome, the player auto-pauses as well.

A better solution could be to trigger the auto-pause only after a certain amount of time, so that short sounds like notifications won't trigger it.

Another solution could be to fadeOut the player's volume during a certain amount of time, then pauses it if the sound that trigger the event is still playing (or fadeIn it again if the sound is not playing anymore).
Like this, the utility should reduce the volume just the time of a notification so you can hear it, or fadeOut and then pauses your player if the sound is too long to be a notification sound.

**Comment by choco** [comment]
Just wanted to add that if I'm not mistaken, OSX notification sound is produced by a system process with other audio effects, it would probably be wise to ignore sound coming from that process when dealing with auto pausing.

**Comment by stewartadam** [comment]
Came here to post the same - even simpler than process exclusions, it would be nice to have a customizable timeout for the auto-pause so that <2s audio from notification chimes doesn't prompt a quick pause-unpause cycle.
